src/policy/policy.h

```diff
@@ -27,6 +27,9 @@ static const unsigned int MAX_P2SH_SIGOPS = 15;
 static const unsigned int MAX_STANDARD_TX_SIGOPS = MAX_BLOCK_SIGOPS/5;
 /** Default for -maxmempool, maximum megabytes of mempool memory usage */
 static const unsigned int DEFAULT_MAX_MEMPOOL_SIZE = 300;
+/** Default for -blocksizeacceptlimit */
```

Comment:
Thanks, will fix.
Its multiples of 100kB because its much easier to write 20 than 2000000. I do think people want to allow things like 1.5MB which would make the finest grained change 100kB.